src/extension.ts

```diff
@@ -494,11 +493,10 @@ async function registerTutorialWithDidact(context: vscode.ExtensionContext, name
 		const didactExt: any = vscode.extensions.getExtension(extensionId);
 		if (didactExt) {
 			const commandId: string = 'vscode.didact.register';
-			const tutorialPath: string = path.join(context.extensionPath, extpath);
-			const tutorialUri: vscode.Uri = vscode.Uri.parse(`file://${tutorialPath}`);
+			const tutorialUri: vscode.Uri = vscode.Uri.file(context.asAbsolutePath(extpath));
 
 			// then pass name, uri, and category
```

Comment:
it is no more the uri which is passed

Reply:
Regardless of the object being passed, the object itself BECOMES a URI object for the tutorial, so I think the name is still appropriate for the variable so I'm not sure why you're arguing over this point.